site/docs/src/commands/normalizer-metrics.md

```diff
@@ -0,0 +1,90 @@
+# normalizer metrics
+
+## Metrics
+
+We count:
+
+- [Object applications](#object-applications)
+- [Object formations](#object-formations)
+- [Dynamic dispatches](#dynamic-dispatches)
+- [Dataless formations](#dataless-formations)
+
+### PHI grammar
+
+![phi-grammar](../media/phi-grammar.png)
+
+### Object formations
+
+- `⟦ d ↦ ∅, c ↦ ∅ ⟧`
+
+### Object applications
+
+- `ξ.b(c ↦ ⟦ ⟧)`
+
+### Dynamic dispatches
+
+- `ξ.ρ.c`
+
+### Dataless formations
+
+- `Primitive formation` - a formation that has a Δ-attribute.
+  - `⟦ Δ ⤍ 00- ⟧`
+- `Dataless formation` - not primitive and does not have attributes bound to primitive formations.
+  - `⟦ d ↦ ⟦ φ ↦ ξ.ρ.c, ν ↦ ⟦ Δ ⤍ 00- ⟧ ⟧, c ↦ ∅ ⟧`
```

Comment:
So just to clarify, we mean it does not have _immediate_ attributes bound to primitive formations, but it's ok if its descendants do?
So the formation bound to `d` would be neither primitive nor dataless, right?

Reply:
Correct.

Reply:
Would be great if you can add this clarification

Reply:
A formation doesn't have non-immediate attributes, does it?